src/Modal.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace atk4\ui;
+
+class Modal extends View
```

Comment:
I'm thinking about making it transparent, e.g.

``` php
$modal->set(function($page) {
   $page->add('LoremIpsum');
});
```

This way you can set callback for a modal and you don't have to create virtual page. One option is to extend VirtualPage (then set() wolud work) but you can also simply define function set().